config.sample.json

```diff
@@ -0,0 +1,5 @@
+{
+    "api_token":"<token_here>",
+    "start_date":"2017-01-01",
+    "page_size":500
+}
```

Comment:
Please at a newline at the end of this file

Reply:
hi @luandy64 , i have tested this code. `self.streams` will refer to the class attribute `streams` declared on the class itself, it might not be a good practice to modify class attributes via instances as it can result in an inconsistent state but we are only creating one instance of this class,

does this need to be changed?

```python
class PipedriveTap(object):
    streams = [
        CurrenciesStream(),
        ActivityTypesStream(),
        StagesStream(),
        ... 
]

    def __init__(self, config, state):
        self.config = self.get_default_config()
        ...

        page_size = int(self.config.get('page_size',500))
        for stream in self.streams:
            stream.limit = stream.next_start = page_size
```